src/screens/PostThread/components/ThreadItemAnchorNoUnauthenticated.tsx

```diff
@@ -0,0 +1,32 @@
+import {View} from 'react-native'
+import {Trans} from '@lingui/macro'
+
+import {atoms as a, useTheme} from '#/alf'
+import {Lock_Stroke2_Corner0_Rounded as LockIcon} from '#/components/icons/Lock'
+import * as Skele from '#/components/Skeleton'
+import {Text} from '#/components/Typography'
+
+export function ThreadItemAnchorNoUnauthenticated() {
+  const t = useTheme()
+
+  return (
+    <View style={[a.p_lg, a.gap_md]}>
+      <Skele.Row style={[a.align_center, a.gap_md]}>
+        <Skele.Circle size={42}>
+          <LockIcon size="md" fill={t.atoms.text_contrast_medium.color} />
+        </Skele.Circle>
+
+        <Skele.Col>
+          <Skele.Text style={[a.text_lg, {width: '20%'}]} />
+          <Skele.Text blend style={[a.text_md, {width: '40%'}]} />
+        </Skele.Col>
+      </Skele.Row>
+
+      <View style={[a.py_sm]}>
+        <Text style={[a.text_xl, a.italic, t.atoms.text_contrast_medium]}>
+          <Trans>You must sign-in to view this post.</Trans>
```

Comment:
```suggestion
          <Trans>You must sign in to view this post.</Trans>
```
nit: I don't think the hyphen is necessary here as it's the verb form, and we don't currently use it in similar strings, e.g.
https://github.com/bluesky-social/social-app/blob/7341294df6156afdf24ab43e1e27ebba94f265ad/src/components/PostControls/PostMenu/PostMenuItems.tsx#L463